operator/gitops/argocd/pipeline-service/pipelines-as-code/bugfix.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pac-secret-manager
+  namespace: pipelines-as-code
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pac-secret-manager
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pac-secret-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pac-secret-manager
+subjects:
+  - kind: ServiceAccount
+    name: pac-secret-manager
+    namespace: pipelines-as-code
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: pac-secret-bugfix
```

Comment:
nits (non-blocking):

- Add a YAML comment linking to the JIRA bug
- Perhaps give it a more descriptive suffix? Ex: `-reaper`, `-pruner`, etc.

Reply:
Fixed.